index.js

```diff
@@ -0,0 +1,14 @@
+import './modules/navigation.js';
+import './modules/awesomeBooks.js';
```

Comment:
- [ ] For the sake of best practice, I will suggest you export the functions in these files and import them instead of importing the entire file. That way, you only import the functions needed and not have unnecessary functions imported along with them when you have large files.